generators/vsix-vs-win/BotBuilderVSIX-V4/source.extension.vsixmanifest

```diff
@@ -10,7 +10,7 @@
         <Tags>Bot, Bots, Bot Framework, BotBuilder, Bot Builder, v4bot, sdkbot, botframework</Tags>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0]" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
```

Comment:
Perfect. This will support all VS2017 versions, and all VS 2019 versions (16.0, 16.1, etc).